engine/objconv/mesher/to_OgreMesh.cpp

```diff
@@ -139,9 +140,9 @@ static string getMaterialHash( const GFXMaterial &mat,
     ret += tmp;
     sprintf( tmp, "%06X", (unsigned int) blend_dst );
     ret += tmp;
-    sprintf( tmp, "%06X", *(unsigned int*) &polygon_offset );
+    sprintf( tmp, "%06X", *(unsigned int*) &polygon_offset ); // Why do we need this weird casting?
```

Comment:
From my local experiments, it seems like a trick to turn a floating-point number into a hex. Since this seems to work consistently, and it is for generating a hash, I think we should keep it as is.